extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -261,7 +265,8 @@ public function getWikiVariables() {
 				'enableDiscussions' => $wgEnableDiscussions,
 				'enableDiscussionsImageUpload' => $wgEnableDiscussionsImageUpload,
 				'enableFandomAppSmartBanner' => !empty( $enableFAsmartBannerCommunity ),
-				'enableFeedsAndPosts' => $wgEnableFeedsAndPostsExt,
+				'enableFeedsAndPostsEmbeddedFeed' => $wgEnableFeedsAndPostsExt &&
```

Comment:
Does any of our front end apps use this endpoint? If not then I would remove this variable from here (as well as few others)